static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,207 @@
+# Data Storage On External Hard Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data are huge, which means that they won't fit on our home
```

Comment:
"Sometimes the data" -> "Some times data"
"data are" -> "data files are"

Reply:
UPDATE: Actually "Sometimes" is correct, but we should still change "are" for "is", as we do in all our existing docs.

![image](https://user-images.githubusercontent.com/1477535/63384276-5ce93500-c353-11e9-9e72-ab9c803f847e.png)
https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular

Reply:
UPDATE: Actually "Sometimes" is correct, but we should still change "are" for "is", as we do in all our existing docs.

![image](https://user-images.githubusercontent.com/1477535/63384276-5ce93500-c353-11e9-9e72-ab9c803f847e.png)
https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular

Reply:
I agree with @jorgeorpinel on this one.

Try to google `data is huge` vs `data are huge`.

data is huge sounds way more natural to me